store/postgres/src/writable.rs

```diff
@@ -63,6 +65,11 @@ impl WritableSubgraphStore {
     fn load_deployment(&self, site: &Site) -> Result<SubgraphDeploymentEntity, StoreError> {
         self.0.load_deployment(site)
     }
+
+    // returns ref of SubgraphStore
+    fn inner(&self) -> &SubgraphStore {
+        &self.0
+    }
```

Comment:
The idea behind `WritableSubgraphStore` was to make it clear what functionality of a `SubgraphStore` gets exposed to the `WritableStore`. Instead of exposing `self.0` wholesale, add method(s) that expose the specific `SubgraphStore` functionality.